Makefile

```diff
@@ -127,13 +128,35 @@ docker-push:
 	@echo "Pushing docker image: $(DOCKER_TAG)"
 	@-docker push "$(DOCKER_TAG)"
 
+.PHONY: docker-stop
+docker-stop:
+	@echo "Stopping test docker image: $(DOCKER_TEST)"
```

Comment:
I think you mean docker container, not docker image here.

Reply:
Fixed